Sources/CNIOSHA1/update_and_patch_sha1.sh

```diff
@@ -41,17 +39,21 @@ for f in sha1.c sha1.h; do
     for func in sha1_init sha1_pad sha1_loop sha1_result; do
         "$sed" -i \
             -e "s/$func/c_nio_$func/g" \
-            -e 's#<sys/systm.h>#<strings.h>#g' \
-            -e 's#<crypto/sha1.h>#"include/CNIOSHA1.h"#g' \
-            -e 's%#ifdef _KERNEL%#define __min_size(x)	static (x)%g' \
-            -e 's%#endif /\* _KERNEL \*/%%g' \
-            -e 's%__FBSDID("$FreeBSD$");%%g' \
             "$here/c_nio_$f"
     done
 done
 
-gsed -i $'/#define _CRYPTO_SHA1_H_/a #ifdef __ANDROID__\\\n#include <sys/endian.h>\\\n#endif' "$here/c_nio_sha1.h"
-gsed -i '/#define _CRYPTO_SHA1_H_/a #include <sys/types.h>' "$here/c_nio_sha1.h"
+gsed -e $'/#define _CRYPTO_SHA1_H_/a #include <stdint.h>\\\n#include <stddef.h>' \
```

Comment:
Drive by comment:

```suggestion
$sed -e $'/#define _CRYPTO_SHA1_H_/a #include <stdint.h>\\\n#include <stddef.h>' \
```

Reply:
Does that matter here? Doesn't `$sed` always refer to GNU sed but it's spelt `gsed` on Darwin and `sed` elsewhere?

Reply:
oh sorry, you're right. Yeah, we need to make this `$sed`